src/public/modules/core/services/gtag.client.service.js

```diff
@@ -354,22 +360,25 @@ function GTag($rootScope, $window) {
     expectedNumSubmissions,
     numWorkers,
   ) => {
+    const userDetails = JSON.parse($window.localStorage.getItem('user'))
```

Comment:
Better to extract this code into a helper function, rather than copying and pasting!

Reply:
See if the changes work!